### PR TITLE
improve layout of developer docs

### PR DESCRIPTION
**Pull request by kambala-decapitator**

### CODE REVIEW

File: docs/developers/Building_Android.md [hunk 1]
```diff
@@ -10,17 +10,17 @@ The following instructions apply to **v1.2 and later**. For earlier versions the
...
+    - download from https://developer.android.com/ndk/downloads
+    - download with Conan, see [#NDK and Conan](#ndk-and-conan)
+5. (optional) Ninja: download from your package manager or from https://github.com/ninja-build/ninja/releases
 
 ## Obtaining source code
 
 Clone https://github.com/vcmi/vcmi with submodules. Example for command line:
 
-```sh
+```
```
Comment (IvanSavenko) [inline docs/developers/Building_Android.md:23]: Don't mind the change, but why?

Comment (kambala-decapitator) [reply docs/developers/Building_Android.md:23]: Doesn't add any value: no bash stuff to highlight

File: client/icons/generate_icns.py
Comment (Nordsoft91) [inline client/icons/generate_icns.py:1]: what is it?

Comment (kambala-decapitator) [reply client/icons/generate_icns.py:1]: if you're asking what the script does, it generates app icon for macOS

if you're asking about the change, it makes the script executable (check the commit message)

**Pull request merged**